lib/client/client_test.go

```diff
@@ -121,7 +121,6 @@ kind: notCalicoApiConfig
 		"APIV1_K8S_CERT_FILE":    "baz1",
 		"APIV1_K8S_KEY_FILE":     "foo1",
 		"APIV1_K8S_CA_FILE":      "foobar1",
-		"APIV1_K8S_API_TOKEN":    "foobarbaz1",
```

Comment:
Suggest you leave the env var _input_ in place (with comment `// ignored for security reasons`) but verify that it is now ignored.

Reply:
Acknowledged and updated code.